### ISSUE TITLE
Accessibility - Add alternative text to main page posters

### ISSUE THREAD

**Issue by wally-yawn** [wally-yawn/rancid-tomatillos#31]
Wave identified that movie posters are lacking alt text, this is a defect that alt is using title which is never passed.
